src/main/java/openblocks/client/bindings/KeyInputHandler.java

```diff
@@ -7,20 +7,35 @@
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.settings.KeyBinding;
 import openblocks.Config;
+import openblocks.common.entity.EntityHangGlider;
 import openblocks.events.PlayerActionEvent;
 import org.lwjgl.input.Keyboard;
 
 public class KeyInputHandler {
 
 	private KeyBinding brickBinding;
+	private KeyBinding varioSwitchBinding;
+	private KeyBinding varioVolUpBinding;
+	private KeyBinding varioVolDownBinding;
 
 	private boolean brickKeyPressed;
+	private boolean varioSwitchKeyPressed;
+	private boolean varioVolUpKeyPressed;
+	private boolean varioVolDownKeyPressed;
 
 	public void setup() {
 		if (!Config.soSerious) {
 			brickBinding = new KeyBinding("openblocks.keybind.drop_brick", Keyboard.KEY_B, "openblocks.keybind.category");
 			ClientRegistry.registerKeyBinding(brickBinding);
 		}
+		if (Config.hanggliderEnableThermal) {
+			varioSwitchBinding = new KeyBinding("openblocks.keybind.vario_switch", Keyboard.KEY_V, "openblocks.keybind.category");
+			varioVolUpBinding = new KeyBinding("openblocks.keybind.vario_vol_up", Keyboard.KEY_I, "openblocks.keybind.category");
+			varioVolDownBinding = new KeyBinding("openblocks.keybind.vario_vol_down", Keyboard.KEY_K, "openblocks.keybind.category");
```

Comment:
Thats true, also it would be more intuitive. On the other hand that will prevent in-flight volume changes.
What's your opinion on using the vanilla controls as master volume but keeping the keybindings (default 'NONE') for players that want to have more separate volume controls (like me)?